package.json

```diff
@@ -51,7 +51,8 @@
     "gulp-footer": "^1.0.5",
     "gulp-header": "^1.7.1",
     "gulp-if": "^2.0.2",
-    "gulp-jsdoc-to-markdown": "^1.2.1",
+    "gulp-documentation": "3.2.1",
+    "documentation": "5.2.2",
```

Comment:
Let's move `documentation` above, to keep this mammoth list of dev-depdendencies alphabetized.

Also: Stick a `^` before the version (like the surrounding ones). That makes it download the latest version, up until the library makes a breaking change that we actually have to pay attention to..

Reply:
Thanks @dbemiller !  Made all requested changes:

- Alphabetize package list
- Add "^" to versions
- Update `yarn.lock`